plugins/org.jkiss.dbeaver.ext.cubrid/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: DBeaver Cubrid Support
 Bundle-SymbolicName: org.jkiss.dbeaver.ext.cubrid;singleton:=true
-Bundle-Version: 1.0.189.qualifier
+Bundle-Version: 1.0.0

```

Comment:
Sorry. Please follow the same format.
1.0.0 -> 1.0.0.qualifier

Reply:
Ok, I will update it now. if there's any remaining mistakes, please let me know.